docs/source/install.rst

```diff
@@ -91,70 +91,35 @@ That shows all logs of installation. It may helps you::
 
 .. _install_cuda:
 
-Install Chainer with CUDA
+Enable CUDA/cuDNN support
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You need to install CUDA Toolkit before installing Chainer.
-If you have CUDA in a default directory or set ``CUDA_PATH`` correctly, Chainer installer finds CUDA automatically::
+In order to enable CUDA support, you first have to install `CuPy <https://docs.cupy-ndarray.org/>`_.
+It requires CUDA Toolkit and ``CUDA_PATH`` environment variable.
```

Comment:
If CuPy's installation guide is well written, we may only need to let users read them.